pages/example_one.py

```diff
@@ -3,8 +3,5 @@
 import streamlit as st
 from page_config import standard_page_widgets
 
+# Add this on top of any page to make mpa-config work!
 standard_page_widgets()
```

Comment:
If we're going to remove these, we should probably put some dummy content in each page, and put this somewhere (probably in the README, maybe also in streamlit_app.py) to point out that you need to run that function in each page.